application/src/main/java/org/opentripplanner/updater/siri/updater/SiriETUpdater.java

```diff
@@ -35,30 +37,27 @@ public class SiriETUpdater extends PollingGraphUpdater {
 
   private final EstimatedTimetableHandler estimatedTimetableHandler;
 
-  private final Consumer<UpdateResult> recordMetrics;
+  private final Consumer<UpdateResult> metricsConsumer;
 
   public SiriETUpdater(
     SiriETUpdaterParameters config,
```

Comment:
This should use the interface not the class.

Reply:
3b06b85bb1df699b5ecb4170a8b9765c880c892a